.github/workflows/test-and-publish-nightly.yml

```diff
@@ -36,10 +36,18 @@ jobs:
         container: ["public.ecr.aws/async-profiler/asprof-builder-x86:latest"]
         include:
           - runson:
-              display: macos-14-arm64
+              display: macos-13
+              name: macos-13
+            java-version: 11
+            java-distribution: corretto
+            architecture: x64
+            container: ""  # Not using container for mac-os as we have images only for linux
+          - runson:
+              display: macos-14
               name: macos-14
             java-version: 11
```

Comment:
Maybe use different JDK versions on x64 and arm64 for wider coverage? E.g. can be 21 on macos-arm64 and 11 on macos-x86.

Reply:
42a2a2f87576bab42bbafbee859c0d924b7f0b59